scipy/sparse/_compressed.py

```diff
@@ -1407,6 +1407,48 @@ def _divide_sparse(self, other):
             return out
 
 
+    def broadcast_to(self, shape):
+        if self.format != 'csr':
+            raise NotImplementedError('broadcasting not implemented for'
+                                      'CSC format.')
+        
+        old_shape = self.shape
+        
+        if old_shape == shape:
+            return self.copy()
```

Comment:
`np.broadcast_to()` returns a view, not a copy. In our case, maybe we could pass `copy=False` as a keyword argument, similar to a few of the other sparse methods.

Reply:
Would that just be something like this - https://github.com/scipy/scipy/pull/21412/commits/a1811faaee68c6de5f2beac6c5a038e2e91dd402?

Reply:
That is close, but it doesn't return a copy when `copy is True`.  How about something like this?
```python
return self.copy() if copy else self
```

Reply:
Addressed at e67d2f7e9776c823400e58a5e3a52ead69efbb25.